.yamato/_projectcontext.yml

```diff
@@ -286,84 +286,84 @@ test_all_project_macos_2020.2:
       -  unity-downloader-cli --source-file unity_revision.txt -c editor --wait --published-only
-      -  upm-ci project test -u .\.Editor --project-path TestProjects/SRP_SmokeTest --type vetting-tests
+      -  upm-ci project test -u .\.Editor --project-path TestProjects/SRP_SmokeTest --type vetting-tests --extra-create-project-arg="-upmNoDefaultPackages"
     artifacts:
         logs:
             paths:
               -  "**/test-results/**"
     variables:
         CUSTOM_REVISION: custom_revision_not_set
-test_all_project_macos_CUSTOM-REVISION:
+test_all_project_OSX_CUSTOM-REVISION:
     name: Test all packages [project context] OSX CUSTOM-REVISION
     agent:
         type: Unity::VM::osx
         image: package-ci/mac:stable
         flavor: m1.mac
     dependencies:
       -  .yamato/_projectcontext.yml#pack_all_project
-      -  .yamato/_editor_priming.yml#editor:priming:CUSTOM-REVISION:macos
+      -  .yamato/_editor_priming.yml#editor:priming:CUSTOM-REVISION:OSX
     commands:
       -  npm install upm-ci-utils@stable -g --registry https://artifactory.prd.cds.internal.unity3d.com/artifactory/api/npm/upm-npm
       -  pip install unity-downloader-cli --index-url https://artifactory.prd.it.unity3d.com/artifactory/api/pypi/pypi/simple --upgrade
       -  unity-downloader-cli --source-file unity_revision.txt -c editor --wait --published-only
-      -  upm-ci project test -u $(pwd)/.Editor/Unity.app/Contents/MacOS/Unity --project-path TestProjects/SRP_SmokeTest --type vetting-tests
+      -  upm-ci project test -u $(pwd)/.Editor/Unity.app/Contents/MacOS/Unity --project-path TestProjects/SRP_SmokeTest --type vetting-tests --extra-create-project-arg="-upmNoDefaultPackages"
     artifacts:
         logs:
             paths:
               -  "**/test-results/**"
     variables:
         CUSTOM_REVISION: custom_revision_not_set
-test_all_project_windows_min_editor:
+test_all_project_Win_min_editor:
     name: Test minimum editor version - all packages [project context] Win
     agent:
         type: Unity::VM
         image: package-ci/win10:stable
         flavor: b1.large
     dependencies:
       -  .yamato/_projectcontext.yml#pack_all_project
-      -  .yamato/_editor_priming.yml#editor:priming:test_min_editor:windows
+      -  .yamato/_editor_priming.yml#editor:priming:test_min_editor:Win
     commands:
       -  npm install upm-ci-utils@stable -g --registry https://artifactory.prd.cds.internal.unity3d.com/artifactory/api/npm/upm-npm
       -  pip install unity-downloader-cli --index-url https://artifactory.prd.it.unity3d.com/artifactory/api/pypi/pypi/simple --upgrade
       -  unity-downloader-cli --source-file unity_revision.txt -c editor --wait --published-only
-      -  upm-ci project test -u .\.Editor --project-path TestProjects/SRP_SmokeTest --type vetting-tests
+      -  upm-ci project test -u .\.Editor --project-path TestProjects/SRP_SmokeTest --type vetting-tests --extra-create-project-arg="-upmNoDefaultPackages"
```

Comment:
For 10.x.x/release I couldn't have this argument to work. I need to make a proper PR to work on it, could you remove it from this one?
Sorry for the confusion, the intent of adding it was good, it's just that 10.x.x/release is not up to date compared to the other branches on this.

Reply:
Ah alright, my bad, pushed a fix!